templates/zerver/integrations/giphy.md

```diff
@@ -1,14 +1,14 @@
-# GIPHY GIF integration
+# Zulip GIPHY GIF integration
 
-Send animated GIFs with your message using GIPHY integration.
+Send animated GIFs with your message using GIPHY.
 
-![](/static/images/integrations/giphy/GIPHY_zulip.png)
+GIPHY is enabled by default in Zulip Cloud. Follow
```

Comment:
Just removed the tabbed section, and made it a single sentence (removed the tip formatting), as recommended by Alya Abbott in [CZO](https://chat.zulip.org/#narrow/channel/19-documentation/topic/integration-docs.3A.20Using.20tabbed.20sections.20for.20a.20single.20instruc/near/1996907).